packages/sdk/src/types/Vouchers.ts

```diff
@@ -251,3 +251,112 @@ export type VouchersBulkUpdateMetadataResponse = {
 export type VouchersBulkUpdateResponse = {
 	async_action_id: string
 }
+
+export interface ListGiftCardTransactionsQuery {
+	limit?: number
+	page?: number
+}
+
+export interface ListGiftCardTransactionsResponse {
+	object: 'list'
+	data_ref: 'data'
+	data: ListGiftCardTransactionsTypeDataObject[] | []
+	total?: number
+	has_more?: boolean
+}
+
+export type ListGiftCardTransactionsType =
+	| 'CREDITS_REDEMPTION'
+	| 'CREDITS_REFUND'
+	| 'CREDITS_ADDITION'
+	| 'CREDITS_REMOVAL'
+
+export interface ListGiftCardTransactionsTypeDataObject {
+	id: string
+	source_id: string | null
+	voucher_id: string
+	campaign_id: string | null
+	source: 'voucherify-web-ui' | 'API' | null
+	reason: string | null
+	type: ListGiftCardTransactionsType
+	details: {
+		balance: {
+			type: 'gift_voucher'
+			total: number
+			amount: number
+			object: 'balance'
+			balance: number
+			operation_type?: 'MANUAL'
+			related_object: {
+				id: string
+				type: 'voucher'
+			}
+		}
+		order?: {
+			id: string
+			source_id: string | null
+		}
+		redemption?: {
+			id: string
+		}
+		rollback?: {
+			id: string
+		}
+	}
+	related_transaction_id: string | null
+	created_at: string
+}
+
+export interface ExportGiftCardTransactionsBody {
+	parameters?: {
+		order?: '-created_at' | 'created_at'
```

Comment:
this shall be equal to ExportGiftCardTransactionsFields and ExportGiftCardTransactionsFields.map(e => `-${e}`)

Reply:
do you mean that each field in ExportGiftCardTransactionsFields can be an `order` value (descending or ascending)?

Reply:
even if you call the export from the V% dashboard the only possible options are '-created_at' | 'created_at'